### ISSUE TITLE
Feature: Create a test file for the current module

### ISSUE THREAD

**Issue by mrjoelkemp** [mrjoelkemp/Dependents#132]
Context menu option for the current file.

We can detect the type of the current module and create a new view with a default test file.

* Assumes that an amd module is using a require mechanism for testing.

Depends on #129 to generate the module name to pull in.

**Comment by mrjoelkemp** [comment]
Closing in favor of https://github.com/mrjoelkemp/node-dependents-editor-backend/issues/29
